<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InAirTable</class>
 <widget class="QDialog" name="InAirTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>681</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>InAirTable</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>470</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;&lt;font size=&quot;+1&quot;&gt;In-air Measurement List&lt;/font&gt;&lt;/b&gt;&lt;/p&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="dataLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>41</y>
     <width>480</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;p align=&quot;center&quot;&gt;Machine&lt;/p&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelFS">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>580</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Field Size</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelLarge">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>620</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Larger:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelSmall">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Smaller:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>630</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDone">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Done</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelAll">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>590</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>All Profiles:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAllOff">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>590</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAllOn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>590</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSmallOff">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLargeOff">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>640</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSmallOn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>600</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLargeOn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>640</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelY">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>614</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>y Profiles:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelx">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>590</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>x Profiles:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonYOn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>614</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelZ">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>639</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>z Profiles:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonZOff">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>639</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonXOff">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonZOn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>639</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonXOn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>590</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonYOff">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>614</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelZ100">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>614</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Z = 100 :</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonZ100On">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>614</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonZ100Off">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>614</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textLabelRect">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>638</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Non-Sq.:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRectOn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>638</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRectOff">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>638</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFS">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>601</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableProfile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>550</width>
     <height>480</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>16</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>70</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>File</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prof</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Z</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Width</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Length</string>
    </property>
   </column>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>InAirTable</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
